packages/explat-client/src/internal/test/experiment-assignment-store.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Internal dependencies
+ */
+import ExperimentAssignmentStore from '../experiment-assignment-store';
+import { validExperimentAssignment, validFallbackExperimentAssignment } from '../test-common';
+
+describe( 'state', () => {
```

Comment:
It'd be good to also add a test for trying to store an invalid experiment assignment.